pyproject.toml

```diff
@@ -0,0 +1,83 @@
+[project]
+name = "HOTOSM Tasking Manager"
+version = "0.1.0"
```

Comment:
What is this the version of? Tasking Manager or the TOML file? If it is Tasking Manager, we could bump this before every release.

Reply:
I will check that out.